### ISSUE TITLE
[JOSS review] Suggestions and requests

### ISSUE THREAD

**Issue by janfreyberg** [Saran-nns/sorn#31]
Hi, I have a couple of suggestions / requests for your documentation. Let me know if you disagree with any of these:

- [x] your documentation references a [configuration file](https://self-organizing-recurrent-neural-networks.readthedocs.io/en/latest/usage.html#update-network-configuration) which lives inside the installed python package, and is suggested to be edited by the user. This doesn't seem like a good solution for configuring the core part of your library, as the site package directory may not always be editable by the user (e.g. if the library was installed by an administrator, the library is shared across users, etc.)

    I would suggest changing the way this is handled. One way would be to define these in code, in a frozen dictionary inside the modules you have elsewhere. You could then allow users to copy this dictionary, edit it, and pass it as an argument to the SORN class.

- [x] It would be great if you could expand the OpenAI gym example to be "self-contained". Currently it references a pickle file which isn't generated in the documentation. If this is intended to be the pkl file in the github repository, this should be explained, because that pkl file doesn't exist if someone installs the library via pip. It also references computing gradients - is it possible to supply a simple example of this?
- [x] There seems to be some discrepancy between the README and the hosted documentation, for example the OpenAI Gym example. It would be nice if they could match (you could even include your README in your documentation itself).
- [x] It would be nice to include some example pictures of the plots in the documentation, so that users know what to expect

- [x] Running test_sorn.py produces some corrupted graphs, e.g.:
    ![image](https://user-images.githubusercontent.com/4092425/129411686-13abc284-392a-4c75-853b-4ddb7547d2b0.png)

**Comment by Saran-nns** [comment]
Hi @janfreyberg . Thank you for the suggestions. 

The `configuration.ini` file is removed. The default values are defined directly at [SORN class](https://github.com/Saran-nns/sorn/blob/4488d9ed52799d2fbae662b454d1a63129240410/sorn/sorn.py#L13). Users can construct the network using the `Simulator` method [simulate_sorn](https://github.com/Saran-nns/sorn#override-default-values-and-simulate-new-sorn-model) or the `Trainer` method [train_sorn](https://github.com/Saran-nns/sorn#training-phase). The examples at the `Usage` section of README are updated accordingly.

OpenAI gym example in the README is updated based on your suggestion.

I have updated the documentation hosted at [readthedocs.io](https://self-organizing-recurrent-neural-networks.readthedocs.io/en/latest/) to match the README now.

Example figures are included in README and documentation at the latest versions

`test_sorn.py` plots are updated and tested

Thank you :)

**Comment by Saran-nns** [comment]
Further, I made some minor changes to` paper.md`. The throw-away variables like `_ne,_nu,_lambda_ee` which were used to obtain network hyperparameters from the `configuration.ini` file are now changed to `ne,nu,lambda_ee`.

**Comment by janfreyberg** [comment]
Thanks, looks good :)
